persistence/registry/persist-via-active-setup-registry-key.yml

```diff
@@ -0,0 +1,19 @@
+rule:
+  meta:
+    name: persist via Active Setup registry key
+    namespace: persistence/registry
+    author: moritz.raabe@fireeye.com
+    scope: function
+    att&ck:
+      - Persistence::Boot or Logon Autostart Execution::Active Setup [T1547.014]
+    references:
+      - https://www.fireeye.com/blog/threat-research/2017/02/spear_phishing_techn.html
+    examples:
+      - c335a9d41185a32ad918c5389ee54235:0x4028F0
+  features:
+    - and:
+      - or:
+        - match: set registry value
+        - number: 0x80000002 = HKEY_LOCAL_MACHINE
```

Comment:
Why is this needed? Doesn't `set registry value` match? 🤔

Reply:
this is a hack to cover more cases, e.g. when there's a wrapper function

Reply:
would it make sense to add this to `set registry value`? 🤔

Reply:
If we add it there we cannot flexibly express setting different root keys anymore.